dev/bots/travis_setup.sh

```diff
@@ -7,7 +7,8 @@ set -x
 
 if [ -n "$TRAVIS" ]; then
   export CLOUDSDK_CORE_DISABLE_PROMPTS=1
-  curl https://sdk.cloud.google.com | bash
+  echo "Installing Google Cloud SDK..."
+  curl https://sdk.cloud.google.com | bash > /dev/null
 fi
```

Comment:
Add an echo when it completes so we'll know if it hangs (as it did in https://github.com/flutter/flutter/issues/10940)